huxley/www/js/components/AdvisorRosterView.js

```diff
@@ -168,10 +170,31 @@ var AdvisorRosterView = React.createClass({
             </Button>
           </td>
         );
+
+        var waiverCheck = '';
```

Comment:
Don't use `var`, use `let` or `const`. Also, this can be a one liner:
```
const waiverCheck = delegate && delegate.waiver_submitted ? '\u2611' : '\u2610';

Reply:
ditto re: optional property access